telegram/start_telegram_bot.js

```diff
@@ -311,7 +312,7 @@ module.exports = (args, cbk) => {
         bot.command('blocknotify', ctx => {
           handleBlocknotifyCommand({
             request,
-            reply: n => ctx.reply(n, markdown),
+            reply: n => ctx.reply(escape(n), markdown),
```

Comment:
I think passing escape at this level is problematic because then you can't put in markdown symbols, they will be escaped, so why even use markdown post mode in the first place?

Reply:
I removed some characters in the escape variable definition at the top like * since they are needed. I see your point though

Reply:
If there is no markdown in the message at all then it can just use default parse mode and not escape anything

Reply:
you're saying just do `ctx.reply(n)` without any markdown at all?

Reply:
yeah exactly, but if you just pass the reply function to the ln-telegram side and the markup and stuff or lack thereof can be handled over there

Reply:
Guess almost every interaction has some markdown symbol and needs escaping for the markdown to work.